swagger-java-sam-client/pom.xml

```diff
@@ -0,0 +1,306 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>swagger-java-sam-client</artifactId>
+    <packaging>jar</packaging>
+    <name>swagger-java-sam-client</name>
+
+    <parent>
+        <artifactId>dockstore</artifactId>
+        <groupId>io.dockstore</groupId>
+        <version>1.5.0-alpha.2-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.swagger</groupId>
+                <artifactId>swagger-codegen-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <inputSpec>https://sam.dsde-prod.broadinstitute.org/api-docs.yaml</inputSpec>
```

Comment:
> Is there a current use case for a FireCloud user interacting with Dockstore without ever having logged in?

My understanding is that Dockstore will replace the FireCloud methods repository, so that in the FireCloud UI they will be eventually fetching and storing all workflows/methods in Dockstore. I thought we needed to support this from the start, but we should clarify with Broad.

Reply:
> This is always reading off the live server, raising the spectre of our builds suddenly failing if their server is upgraded or offline. Is this checked in and tagged anywhere in their repos?

It is checked in, but unfortunately they don't tag their releases. I'm worried that we could end up generating a client for something that hasn't yet been deployed. We could broach this at the next meeting with FireCloud.

Reply:
Probably worthwhile to bring it up, being able to say which version of sam we tested against is import. In the meantime, you can point at checked in content by commit
https://help.github.com/articles/getting-permanent-links-to-files/#press-y-to-permalink-to-a-file-in-a-specific-commit

Reply:
OK, changing to point to a GitHub commit in next push.